src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -167,8 +172,13 @@ private void OnRunClicked(object sender, RoutedEventArgs e)
             }
         }
 
-        #endregion
 
+        #endregion
 
+        private void OnMoreInfoClicked(object sender, RoutedEventArgs e)
+        {
+            var viewEx = (WpfUtilities.FindUpVisualTree<DynamoView>(nodeView) as DynamoView).viewExtensionManager.ViewExtensions.FirstOrDefault(x => x.UniqueId == "1f8146d0-58b1-4b3c-82b7-34a3fab5ac5d");
+            (viewEx as PythonMigrationViewExtension).OpenPythonMigrationWarningDocumentation();
```

Comment:
It would be nice if we could avoid the reference to the extension, finding it by id, and then directly calling a method on it. Can we do something similar to what the migration extension is doing?

`LoadedParams.ViewModelCommandExecutive.OpenDocumentationLinkCommand(link);`

Reply:
@mmisol Even we try to get `ViewModelCommandExecutive` the actual hyper link to open will be only accessible from view extension still. Then we still need a reference to `PythonMigrationViewExtension`. That's why I left this code unchanged for now.

I also think through the event based way, but that will require `PythonMigrationViewExtension` subscribe to every python node in new workspace plus every new python new added to the workspace. That still does not seem ideal to me.

One thing @mjkkirschner remind me is that in previous development, we have implemented something like a broadcast mechanism for a center model to broadcast action message to all extensions and only the extension meeting criteria will react to that. That design may fit our case here, but I feel it is out of the scope.

On the other hand, I still hold my original opinion that we may want to move the PythonEditor, Python3 Engine + PythonMigrationViewExtension into a single view extension and make it optional from Dynamo. That seems to be the cleanest way I can see without introducing new mechanism.

Reply:
The way the documentation browser is designed, it can load content from any loaded assembly. You can take a look at the readme here https://github.com/DynamoDS/Dynamo/tree/master/src/DocumentationBrowserViewExtension